src/index.html

```diff
@@ -9,10 +9,54 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/index.scss"
```

Comment:
The `href` attribute should point to a `.css` file instead of a `.scss` file, as browsers cannot directly interpret SCSS files. Please ensure that the SCSS is compiled into CSS and update the link accordingly.